homeassistant/components/devolo_home_network/entity.py

```diff
@@ -32,8 +32,8 @@
 )
 
 
-class DevoloEntity(CoordinatorEntity[DataUpdateCoordinator[_DataT]]):
-    """Representation of a devolo home network device."""
+class DevoloCoordinatorEntity(CoordinatorEntity[DataUpdateCoordinator[_DataT]]):
```

Comment:
DevoloCoordinatorEntity could use both CoordinatorEntity and DevoloEntity to offset some of the duplicating code perhaps (assuming based on your comments from the PR)

Reply:
I had that idea, too but then was unsure which version is better: copying the code or not to rely on the MRO. I checked other integrations and calling init methods explicitly seems to be ok. I adapted accordingly.